tests/Public/New-GuestConfigurationPolicy.Tests.ps1

```diff
@@ -684,6 +684,12 @@ Describe 'New-GuestConfigurationPolicy' {
 
                         $result.PolicyId | Should -Not -BeNullOrEmpty
                         $result.PolicyId.GetType().Name | Should -Be 'Guid'
+
+                        $fileContent = Get-Content -Path $result.Path -Raw
```

Comment:
This is the wrong place for this check.
This specific test is only for result object returned by the function.
The following test is for the policy definition file generated.
Please update that test rather than extracting the file here